src/models/location.model.js

```diff
@@ -52,7 +52,11 @@ const locationSchema = new Schema({
 
 locationSchema.set('toJSON', {
   getters: true,
-  virtuals: true
+  virtuals: true,
+  transform: (doc, ret) => {
+    delete ret.id
```

Comment:
From the [documentation](http://mongoosejs.com/docs/guide.html#id):

Mongoose assigns each of your schemas an id virtual getter by default which returns the documents _id field cast to a string, or in the case of ObjectIds, its hexString.

Basically, the one that is required for documents is `_id`. `id` is just a getter.